src/lib/utilities/route-for-api.ts

```diff
@@ -35,7 +35,7 @@ const replaceNamespaceInApiUrl = (
   return '';
 };
 
-const base = (namespace?: string): string => {
+export const base = (namespace?: string): string => {
   let baseUrl = '';
   const webUrl: string | undefined = get(page).data?.webUrl;
```

Comment:
I'm tempted to grab it from the `window` object so that it works anywhere, but I'm not sure what kind of access Vite/SvelteKit gives us off the top of my head. I _do_ think we do it elsewhere in the codebase, so there is likely prior art.